src/common/components/registerAccountModal/registerAccountModal.tpl.html

```diff
@@ -31,9 +31,8 @@ <h3 translate>Create Your Account</h3>
   ></sign-in-modal>
   <sign-up-modal
     ng-switch-when="sign-up"
-    on-state-change="$ctrl.stateChanged(state)"
-    on-success="$ctrl.onIdentitySuccess()"
-    on-failure="$ctrl.onIdentityFailure()"
+    on-sign-up="$ctrl.onSignUp(donorDetails)"
+    on-sign-in="$ctrl.onSignIn()"
```

Comment:
You also need to update the file `src/common/services/session/sessionModal.tpl.html` as signUpModal is also used there

Reply:
Good catch, that should be fixed now